qiime2/core/type/signature.py

```diff
@@ -401,6 +401,11 @@ def transform_and_add_callable_args_to_prov(self, provenance,
                     self._transform_and_add_input_to_prov(
                         provenance, name, spec, arg)
             else:
+                if spec.qiime_type.name == 'Capture':
```

Comment:
I think we might look at the view type (for `CaptureHolder`) instead if there's no semantic type needed.